en/core-api/cardless-credit.md

```diff
@@ -1,12 +1,11 @@
-# Cardless Credit
+# Cardless Credit Payment Integration
 <hr>
-One of the payment method offered by Midtrans is Cardless Credit. By using this payment method, customers will have the option to make a payment via Cardless credit page and Midtrans will send real time notification when the customer complete the payment.
+<i>Cardless Credit</i> is one of the payment methods offered by Midtrans. Using this payment method, the customers can shop online and make payments in installments. Midtrans sends real-time notifications to you when the customer completes the payment.
```

Comment:
`<i>Cardless Credit</i>` Italic can be done using Markdown formatting, please use `*Cardless Credit*` instead of using `<i>` html tag.

Reply:
In the original document, this introductory paragraph was written inside HTML. So when we edited the content, we kept it the same. However, inside the HTML tag, HTML tags are the only way to make the text appear as Italics. Everywhere else we have used Markdown formatting.

Reply:
hmm it should not be in HTML, can you add new line between `<hr>` and this paragraph, it will make the paragraph become a markdown format separated from the HTML. thanks

Reply:
Ok. I will do that.